### PR TITLE
feat: Display Contacts in the CozyBar on smalscreen

### PR DESCRIPTION
**Pull request by Crash--**
capture d ecran 2019-01-23 a 08 55 02

### CODE REVIEW

**Comment by Crash--** [comment]
[Contacts - cozy-bar tablette/mobile : afficher titre app Contacts](https://trello.com/c/0KI9K9Ho/1523-contacts-cozy-bar-tablette-mobile-afficher-titre-app-contacts)

File: src/components/App.jsx [hunk 1]
```diff
@@ -28,10 +32,21 @@ class ContactsApp extends React.Component {
...
+    const {
+      t,
+      breakpoints: { isMobile }
+    } = this.props
 
     return (
       <Layout monocolumn="true">
+        {isMobile && (
+          <BarCenter>
+            {' '}
```
Comment (y-lohse) [inline src/components/App.jsx]: Why do we need this?

File: src/styles/contacts.styl [hunk 2]
```diff
@@ -120,4 +120,13 @@ $reset-list
 
 .contacts-empty-soon
     display flex
-    justify-content center
\ No newline at end of file
+    justify-content center
+
+
+.fil-path-title
```
Comment (y-lohse) [inline src/styles/contacts.styl]: None of the [cozy-ui text components](https://docs.cozy.io/cozy-ui/react/#text) can do the job? [Or the utility classes](https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html)?

If not, are we missing a size in our component list?

File: src/components/App.jsx (hunk 1 above)
Comment (Crash--) [reply src/components/App.jsx]: Prettier.... Thank you

File: src/styles/contacts.styl (hunk 2 above)
Comment (Crash--) [reply src/styles/contacts.styl]: I used the title component at the end. I'll check if i can remove some css properties. I just c/p from drive to be franc...

**Pull request merged**